docs/Examples.md

```diff
@@ -266,11 +266,55 @@ Now run the below request multiple times and see the requests being load balance
 kubectl exec --namespace=sample -it $(kubectl get pod -l "app=webapp" --namespace=sample -o jsonpath='{.items[0].metadata.name}') -c webapp -- curl -v http://default.greeting.global
 ```
 
+#### Argo-Rollouts 
+
+#### Install Argo-rollouts
+Refer [this](https://argoproj.github.io/argo-rollouts/) for details on Argo-Rollouts
+
+```
+kubectl create namespace argo-rollouts
+
+kubectl apply -n argo-rollouts -f https://raw.githubusercontent.com/argoproj/argo-rollouts/stable/manifests/install.yaml
+```
+
+#### Deploy Sample Argo-Rollouts Services with Blue-Green stratergy
+Refer [this](https://argoproj.github.io/argo-rollouts/features/bluegreen/) for details on Blue-Green stratergy with Argo-Rollouts
+
+```
+#Install test services
+
+kubectl apply -f ./admiral-install-v0.1-beta/yaml/sample-greeting-rollout-bluegreen.yaml
```

Comment:
Replace `./admiral-install-v0.1-beta` with `$ADMIRAL_HOME` You should have this reference after a rebase with master.

Reply:
Done